1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+---
+post_title: Quick Start
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+
+# Launch a long running interactive Bash session
+
+In this example, a long running job app is launched and then the `dcos exec` command is used to launch an interactive Bash session inside of the task container on the node.
+
+1.  Deploy and run a job app with the DC/OS CLI:
+
+    1.  Create the following app definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+          "id": "task-exec-test",
+          "labels": {},
+          "run": {
+            "artifacts": [],
+            "cmd": "sleep 100000000",
+            "cpus": 0.01,
+            "disk": 0,
+            "env": {},
+            "maxLaunchDelay": 3600,
+            "mem": 32,
+            "placement": {
+              "constraints": []
+            },
+            "restart": {
+              "policy": "NEVER"
+            },
+            "volumes": []
+          }
+        }
+        ```
+    
+    1.  Deploy the job app with this CLI command:
+    
+        ```bash
+        $ dcos job add test-job.json
+        ```
+        
+    1.  Verify that the app has been successfully deployed:
+    
+        ```bash
+        $ dcos job list
+        ID              DESCRIPTION  STATUS   LAST SUCCESFUL RUN        
+        task-exec-test               Unscheduled         None
+        ```
+
+    1.  Run the job:
+    
+        ```bash
+        $ dcos job run task-exec-test
+        ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME                                HOST       USER  STATE  ID                                                                       
+    20161209183121nz2F5.klueska-test    10.0.2.53  root    R    <task_id>
+    ```
+
+1.  Launch a TTY process inside of the container with the task ID (`<task_id>`) specified. This will launch an interactive Bash session.
```

Comment:
I would reword this as:
```
Launch a process inside of the container with Task ID (`<task_id>`) and attach a TTY to it. This will launch an interactive Bash session.
```